### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by monamo19**
added notebook for Amazon A2I Fraud detector blog

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Amazon A2I Fraud Detector.ipynb [hunk 1]
```diff
@@ -0,0 +1,796 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "Amazon Augmented AI (Amazon A2I) makes it easy to build the workflows required for human review of ML predictions. Amazon A2I brings human review to all developers, removing the undifferentiated heavy lifting associated with building human review systems or managing large numbers of human reviewers. \n",
+    "\n",
+    "Amazon A2I provides built-in human review workflows for common machine learning use cases, such as content moderation and text extraction from documents, which allows predictions from Amazon Rekognition and Amazon Textract to be reviewed easily. You can also create your own workflows for ML models built on Amazon SageMaker or any other tools. Using Amazon A2I, you can allow human reviewers to step in when a model is unable to make a high confidence prediction or to audit its predictions on an on-going basis. Learn more here: https://aws.amazon.com/augmented-ai/\n",
+    "\n",
+    "In this tutorial, we will show how you can use Amazon A2I directly within your API calls to Amazon Rekognition's Detect Moderation Labels API. \n",
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:46]: This section needs to be changed to reflect fraud detector use case 

"In this tutorial, we will show how you can use Amazon A2I directly within your API calls to Amazon Rekognition's Detect Moderation Labels API."

File: Amazon A2I Fraud Detector.ipynb [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+    "\n",
+    "* A **human review workflow**, also referred to as a flow definition. You use the flow definition to configure your human workforce and provide information about how to accomplish the human review task. For built-in task types, you also use the flow definition to identify the conditions under which a review human loop is triggered. For example, with Amazon Rekognition can perform image content moderation using machine learning. You can use the flow definition to specify that an image will be sent to a human for content moderation review if Amazon Rekognition's confidence score output is low for your use case. You can create a flow definition in the Amazon Augmented AI console or with the Amazon A2I APIs. To learn more about both of these options, see https://docs.aws.amazon.com/sagemaker/latest/dg/a2i-create-flow-definition.html\n",
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:59]: Please remove the reference to Rekognition similar to https://github.com/monamo19/amazon-a2i-sample-jupyter-notebooks/blob/master/Amazon%20A2I%20with%20Amazon%20SageMaker%20for%20object%20detection%20and%20model%20retraining.ipynb

File: Amazon A2I Fraud Detector.ipynb [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+    "NOTE: The following template models a \"Claim\" - i.e. mark if a given claim is fraudulent, valid claim or needs further investigation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template=\"\"\"<script src=\"https://assets.crowd.aws/crowd-html-elements.js\"></script>\n",
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:465]: This should be moved to Create Human Task UI section

File: Amazon A2I Fraud Detector.ipynb [hunk 4]
```diff
@@ -0,0 +1,796 @@
...
+    "        HumanTaskUiName=task_ui_name,\n",
+    "        UiTemplate={'Content': template})\n",
+    "    return response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an Augmented AI task UI"
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:526]: We should include this at top

File: Amazon A2I Fraud Detector.ipynb [hunk 5]
```diff
@@ -0,0 +1,796 @@
...
+    "taskUIName = 'fraud'+ str(uuid.uuid1())\n",
+    "\n",
+    "# Create task UI\n",
+    "humanTaskUiResponse = create_task_ui(taskUIName, template)\n",
+    "humanTaskUiArn = humanTaskUiResponse['HumanTaskUiArn']\n",
+    "print(humanTaskUiArn)"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:545]: Lets also add a block indicating flow definition creation

File: Amazon A2I Fraud Detector.ipynb [hunk 6]
```diff
@@ -0,0 +1,796 @@
...
+    "print(flowDefinitionArn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Detect Moderation Labels with AWS Rekognition\n",
+    "\n",
+    "Let's call Amazon Rekognition to detect moderation labels on the sample images stored in S3 based on the steps above."
```
Comment (sindhuchejerla) [inline Amazon A2I Fraud Detector.ipynb:599]: Please remove this and all references to Rekognition

File: Amazon A2I Fraud Detector.ipynb (hunk 1 above)
Comment (monamo19) [reply Amazon A2I Fraud Detector.ipynb:46]: resolved in new pull request

File: Amazon A2I Fraud Detector.ipynb [hunk 7]
```diff
@@ -0,0 +1,796 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Latest SDK"
+   ]
+  },
+  {
+   "cell_type": "code",
```
[the remark this answers is not in the document]
Comment (monamo19) [reply Amazon A2I Fraud Detector.ipynb:74]: added in new pull request

File: Amazon A2I Fraud Detector.ipynb (hunk 3 above)
Comment (monamo19) [reply Amazon A2I Fraud Detector.ipynb:465]: moved

File: Amazon A2I Fraud Detector.ipynb (hunk 6 above)
Comment (monamo19) [reply Amazon A2I Fraud Detector.ipynb:599]: incorporated in new pull request

**Comment by monamo19** [comment]
I am creating a new pull request and have incorporated all the existing comments.

**Comment by monamo19** [comment]
closing this and creating a new one.

**Pull request closed without merging**